src/Parts/Channel.php

```diff
@@ -29,7 +29,7 @@ class Channel
     public ?int $user_limit;
     public ?int $rate_limit_per_user;
     /**
-     * @var \Ragnarok\Fenrir\Parts\User[]
+     * @var User[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)